### DISCUSSION TITLE
Manual trigger for update resync in Kubernetes operator

### DISCUSSION THREAD

**Discussion by rounakdatta** [Infisical/infisical#6962] [Feature Request]
### Feature description
As far as I explored, there's no provision to manually trigger a resync for the Kubernetes operator. While the `resyncInterval` is good enough for our use case, we wanted to have a manual way of triggering a sync via the dashboard.

### Why would it be useful?
We'd be setting up Infisical in a way that there'd be a huge number of `InfisicalSecret` instances in a single cluster, and thereby all of them handled by a single operator pod. Hence, to reduce serving load on our self-hosted Infisical server, we want to have a big value for `resyncInterval`. Also, we'd not push out updates to secrets very often, however when we do, we'd like it to be reflected soon. We think having an option for manual trigger on the dashboard is a nice way to go about it.

**Comment by JanHolger** [comment]
There could probably be a webhook service for triggering it automatically when the secrets change, similar to how ArgoCD does it with GitOps.
